Source/Plugin.BLE/Shared/Contracts/IDevice.cs

```diff
@@ -77,7 +78,7 @@ public interface IDevice : IDisposable
         /// A task that represents the asynchronous read operation. The Result property will contain a boolean that indicates if the update was successful.
         /// The Task will finish after the RSSI has been updated.
         /// </returns>
-        Task<bool> UpdateRssiAsync();
+        Task<bool> UpdateRssiAsync(CancellationToken cancellationToken = default);
```

Comment:
This change causes a warning on line 25 of this file (and also in line 75 of DeviceBase.cs):
```
XML comment has cref attribute 'UpdateRssiAsync()' that could not be resolved
```
I think the `cref` also needs to include the new parameter (or rather: it's type), probably like this:
```
cref="UpdateRssiAsync(CancellationToken)"
```